modules/edu.gemini.seqexec.model/src/test/scala/edu/gemini/seqexec/model/BoopicklingSpec.scala

```diff
@@ -11,13 +11,14 @@ import org.scalatest.prop.PropertyChecks
 // Keep the arbitraries in a separate trait to improve caching
 object SharedModelArbitraries {
   import org.scalacheck.Shapeless._
-  val maxListSize = 5
+  val maxListSize = 4
 
   // N.B. We don't want to auto derive this to limit the size of the lists for performance reasons
   def sequencesQueueArb[A](implicit arb: Arbitrary[A]): Arbitrary[SequencesQueue[A]] = Arbitrary {
     for {
-      a <- Gen.listOfN[A](maxListSize, arb.arbitrary)
-    } yield SequencesQueue(a)
+      a <- implicitly[Arbitrary[Conditions]].arbitrary
```

Comment:
Not too good coverage really but it is not very critical and don't want very long builds